server/events/output_updater.go

```diff
@@ -57,22 +59,54 @@ func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand
 		templateOverrides = repoCfg.TemplateOverrides
 	}
 
+	// Assue error or failure if empty project results
+	if len(res.ProjectResults) == 0 {
```

Comment:
Why is empty project results considered an error? `command.Result` has two fields for errors `Error` and `Failure`.